registration_ref/app.py

```diff
@@ -46,6 +46,9 @@ def create_in_foundries(client_cert: str, api_token: str, name: Optional[str] =
         "OSF-TOKEN": api_token,
     }
     for x in (0.1, 0.2, 1, 0):
+        log.info("Initiating request")
+        log.info("Using DEVICE_REGISTRATION_API: %s", Settings.DEVICE_REGISTRATION_API)
+        log.info("Using VERIFY_SSL: %s", Settings.VERIFY_SSL)
         r = requests.put(
             "https://api.foundries.io/ota/devices/", headers=headers, json=data
```

Comment:
now we just need to use the url instead of the hard coded url in the call to requests.put

Reply:
yep, that's what I tried to ask for... see this:
> Don't you want to use these new settings here?

Reply:
OHHH that is a mistake.
I did the changes in another clone where I was testing it out, and I missed copying the line.
Good catcher. 

![Screenshot 2024-01-18 at 20 37 15](https://github.com/foundriesio/factory-registration-ref/assets/7708031/1d7cdb26-bfae-42ef-84f1-e5278b65447b)